rst_files/version_control.rst

```diff
@@ -84,17 +85,15 @@ Commits
 
 GitHub stores history as a sequence of changes to text, called *commits* 
 
-Here is an example of a commit, which revises the style guide in a QuantEcon repo (`link <https://github.com/QuantEcon/lecture-source-jl/commit/ba59c3ea9a0dec10def3f4f3928af5e2827f3b92/>`_)
+`Here <https://github.com/QuantEcon/lecture-source-jl/commit/ba59c3ea9a0dec10def3f4f3928af5e2827f3b92/>`_
+ is an example of a commit, which revises the style guide in a QuantEcon repo
 
 In particular, commits have the following features
 
 * An ID (formally, an "SHA-1 hash")
 * Content (i.e., a before and after state)
 * Metadata (author, timestamp, commit message, etc.)
 
-It is crucial to remember that commits represent differences in text, as opposed to repository states 
```

Comment:
I think we need to keep this, since this is consistently one of the most confusing things about git

Reply:
Can you clarify what this means?

Reply:
Basically the idea is that a commit is less like a point in time (say `1`), and more like a move between points in time (say `1 --> 2`). So commits aren't versions, but are rather actions.